examples/with-react-hook-form/src/App.tsx

```diff
@@ -0,0 +1,31 @@
+import { Flex } from "@hover-design/react";
+import "./App.css";
+import Hero from "./components/Hero";
+
+import TextInput from "./components/form-components/text-input/TextInput";
+import SubmitButton from "./components/form-components/button-component/SubmitButton";
```

Comment:
Can you create a `index.ts` file and export all the components from there? Imports will look good!

Reply:
on it